plugins/tracing/client/http.go

```diff
@@ -14,12 +14,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package tracing
+package client
```

Comment:
will this become a problem when we add more clients?  e.g. name conflict, client.Transport is required for both http and other protocol.

/cc @kdada

Reply:
We can use the protocol name as a prefix

Reply:
`plugins/tracing/clients/http/client.go` is more flexible.

Reply:
> plugins/tracing/clients/http/client.go is more flexible.

SGTM